.vscode/tasks.json

```diff
@@ -1,56 +1,66 @@
 {
   "version": "2.0.0",
   "tasks": [
-      {
-          "label": "setup",
-          "command": "npm run dev-setup",
-          "type": "shell"
+    {
+      "type": "typescript",
+      "tsconfig": "tsconfig.json",
+      "option": "watch",
+      "problemMatcher": [
+        "$tsc-watch"
+      ],
+      "group": {
+        "kind": "build",
+        "isDefault": true
```

Comment:
watch doesn't need to be default build option.  nothing we are doing, atm, requires immediate compilation through watch.